cameraview/src/main/java/com/otaliastudios/cameraview/engine/Camera2Engine.java

```diff
@@ -1252,6 +1261,33 @@ public void setPlaySounds(boolean playSounds) {
         mPlaySoundsOp.end(null);
     }
 
+    @Override public void setPreviewFrameRate(float previewFrameRate) {
+        mPreviewFrameRate = previewFrameRate;
```

Comment:
Can you save the old value here like the other setters? Then pass it to applyPreviewFrameRate(). This way the apply function can revert if needed.